packages/neon_lints/lib/dart.yaml

```diff
@@ -9,219 +9,219 @@ analyzer:
-    prefer_interpolation_to_compose_strings: true
-    prefer_is_empty: true
-    prefer_is_not_empty: true
-    prefer_is_not_operator: true
-    prefer_iterable_whereType: true
-    prefer_mixin: true
-    prefer_null_aware_method_calls: true
-    prefer_null_aware_operators: true
-    prefer_relative_imports: false
-    prefer_single_quotes: true
-    prefer_spread_collections: true
-    prefer_typing_uninitialized_variables: true
-    prefer_void_to_null: true
-    provide_deprecation_message: true
-    public_member_api_docs: true
-    recursive_getters: true
-    require_trailing_commas: true
-    secure_pubspec_urls: true
-    sized_box_for_whitespace: true
-    sized_box_shrink_expand: true
-    slash_for_doc_comments: true
-    sort_child_properties_last: true
-    sort_constructors_first: true
-    sort_pub_dependencies: true
-    sort_unnamed_constructors_first: true
-    test_types_in_equals: true
-    throw_in_finally: true
-    tighten_type_of_initializing_formals: true
-    type_annotate_public_apis: true
-    type_init_formals: true
-    type_literal_in_constant_pattern: true
-    unawaited_futures: true
-    unnecessary_await_in_return: true
-    unnecessary_brace_in_string_interps: true
-    unnecessary_breaks: true
-    unnecessary_const: true
-    unnecessary_constructor_name: true
-    unnecessary_final: false
-    unnecessary_getters_setters: true
-    unnecessary_lambdas: true
-    unnecessary_late: true
-    unnecessary_library_directive: true
-    unnecessary_new: true
-    unnecessary_null_aware_assignments: true
-    unnecessary_null_aware_operator_on_extension_on_nullable: true
-    unnecessary_null_checks: true
-    unnecessary_null_in_if_null_operators: true
-    unnecessary_nullable_for_final_variable_declarations: true
-    unnecessary_overrides: true
-    unnecessary_parenthesis: true
-    unnecessary_raw_strings: true
-    unnecessary_statements: true
-    unnecessary_string_escapes: true
-    unnecessary_string_interpolations: true
-    unnecessary_this: true
-    unnecessary_to_list_in_spreads: true
-    unreachable_from_main: true
-    unrelated_type_equality_checks: true
-    unsafe_html: true
-    use_build_context_synchronously: true
-    use_colored_box: true
-    use_decorated_box: true
-    use_enums: true
-    use_full_hex_values_for_flutter_colors: true
-    use_function_type_syntax_for_parameters: true
-    use_if_null_to_convert_nulls_to_bools: true
-    use_is_even_rather_than_modulo: true
-    use_key_in_widget_constructors: true
-    use_late_for_private_fields_and_variables: true
-    use_named_constants: true
-    use_raw_strings: true
-    use_rethrow_when_possible: true
-    use_setters_to_change_properties: true
-    use_string_buffers: true
-    use_string_in_part_of_directives: true
-    use_super_parameters: true
-    use_test_throws_matchers: true
-    use_to_and_as_if_applicable: true
-    valid_regexps: true
-    void_checks: true
+    - always_declare_return_types: true
```

Comment:
Does this work with the `-`? I better like it without :)

Reply:
I can have a look but I'm not sure